shell/key-bindings.bash

```diff
@@ -51,15 +51,16 @@ __fzf_history__() (
   shopt -u nocaseglob nocasematch
   line=$(
     HISTTIMEFORMAT= history |
-    eval "$(__fzfcmd) +s --tac +m -n2..,.. --tiebreak=index --toggle-sort=ctrl-r $FZF_CTRL_R_OPTS" |
-    command grep '^ *[0-9]') &&
-    if [[ $- =~ H ]]; then
-      sed 's/^ *\([0-9]*\)\** .*/!\1/' <<< "$line"
-    else
-      sed 's/^ *\([0-9]*\)\** *//' <<< "$line"
-    fi
+    eval "$(__fzfcmd) +s --tac +m -n2..,.. --tiebreak=index --toggle-sort=ctrl-r $FZF_CTRL_R_OPTS --query=$1" |
+    command grep '^ *[0-9]') && sed 's/^ *\([0-9]*\)\** *//' <<< "$line"
```

Comment:
Does this work with multi-line commands?

- https://github.com/junegunn/fzf/commit/b4a05ff27ed8b1371dc3745ce02f48430345761b
- https://github.com/junegunn/fzf/issues/286